### PR TITLE
Document Raven studio access for RavenDB 5

### PR DESCRIPTION
**Pull request by andreasohlund**

### CODE REVIEW

File: servicecontrol/audit-instances/creating-config-file.md [hunk 1]
```diff
@@ -245,9 +245,24 @@ Type: string
...
+
+### Version 4.26 and above
+
+Browse: 
+
+```no-highlight
+http://localhost:{configured ServiceControl instance maintenance port}
+```
+
+to access the internal database via [the RavenDB studio interface](https://ravendb.net/docs/article-page/5.4/csharp/studio/overview).
```
Comment (andreasohlund) [inline servicecontrol/audit-instances/creating-config-file.md:259]: Do we want to mention maintenance mode here as well?

File: servicecontrol/audit-instances/creating-config-file.md [hunk 2]
```diff
@@ -245,9 +245,24 @@ Type: string
 
 ## Troubleshooting
 
-#### ServiceControl.Audit/ExposeRavenDB
 
-ServiceControl Audit stores its data in a RavenDB embedded instance. By default, the RavenDB instance is accessible only by the ServiceControl Audit service. If, during troubleshooting, direct access to the RavenDB instance is required while ServiceControl Audit is running, ServiceControl Audit can be configured to expose the RavenDB studio.
+ServiceControl Audit stores its data in a RavenDB embedded instance. If, during troubleshooting, direct access to the RavenDB instance is required while ServiceControl Audit is running use the following instructions.
+
+### Version 4.26 and above
```
Comment (WojcikMike) [inline servicecontrol/audit-instances/creating-config-file.md:251]: Do we want to mention here, ner persistance? As you can have the old instance upgraded to the 4.26

Comment (andreasohlund) [reply servicecontrol/audit-instances/creating-config-file.md]: Good catch, I have reworked the sections to convey this now

File: servicecontrol/audit-instances/creating-config-file.md [hunk 3]
```diff
@@ -245,14 +245,29 @@ Type: string
...
-ServiceControl Audit stores its data in a RavenDB embedded instance. By default, the RavenDB instance is accessible only by the ServiceControl Audit service. If, during troubleshooting, direct access to the RavenDB instance is required while ServiceControl Audit is running, ServiceControl Audit can be configured to expose the RavenDB studio.
+ServiceControl Audit stores its data in a RavenDB embedded instance. If, during troubleshooting, direct access to the RavenDB instance is required while ServiceControl Audit is running use the following instructions.
 
 NOTE: [Maintenance mode](/servicecontrol/audit-instances/maintenance-mode.md) is the recommended way to review documents in the embedded RavenDB instance.
 
 WARNING: The ServiceControl RavenDB embedded instance is used exclusively by ServiceControl Audit and is not intended for external manipulation or modifications.
 
+### RavenDB 5
+
+If on Version 4.26 and using [the new RavenDB 5 persister](/servicecontrol/upgrades/new-persistence.md) browse to:
```
Comment (mikeminutillo) [inline servicecontrol/audit-instances/creating-config-file.md:257]:
```suggestion
For instances running version 4.26 and above, which are configured to use [the new RavenDB 5 persistence](/servicecontrol/upgrades/new-persistence.md) browse to:
```

File: servicecontrol/audit-instances/creating-config-file.md [hunk 4]
```diff
@@ -245,14 +245,29 @@ Type: string
...
+
+```no-highlight
+http://localhost:{configured ServiceControl instance maintenance port}
+```
+
+to access the internal database via [the RavenDB studio interface](https://ravendb.net/docs/article-page/5.4/csharp/studio/overview).
+
+### RavenDB 3.5
+
+If on Version 4.25 and below or using the old RavenDB 3.5 persister the ServiceControl Audit instance can be configured to expose the RavenDB studio.
```
Comment (mikeminutillo) [inline servicecontrol/audit-instances/creating-config-file.md:267]:
```suggestion
For instances running version 4.25 and below or using the old RavenDB 3.5 persistence the ServiceControl Audit instance can be configured to expose the RavenDB studio.
```

**Review by mikeminutillo — APPROVED** [review]

**Pull request merged**